data/hai/hai missions.txt

```diff
@@ -209,6 +209,79 @@ mission "Ask the Hai about the Unfettered"
+					to display
+						"reputation: Hai" < 1
+				`	"Why should I get in trouble for defending myself against Sympathizers?"`
+					to display
+						"reputation: Hai" >= 1
+			action
+				set "Hai Sympathizer Warning: Explained Trouble"
+			`	He shrugs. "You should try not to impart human ethical standards onto Hai. The simple fact of the matter is that they are citizens, and you are not.`
+			`	"When a report is made that you have destroyed the vessel of a Hai citizen, it is up to the local security staff to verify the chain of events. If they report it was your fault, then it was your fault, unless you can challenge that in a court. However, you are not a citizen, so the security apparatus can simply brand you as a pirate and prevent you from ever gaining access to a court to challenge it in the first place. Even if you managed to do so, you're a captain; your business is in space. You would be grounded for months to work through the court proceedings to challenge it.`
+			`	"Whether you believe it is right or not, it is simply the reality of the situation. You must manage your guns and your reputation carefully where interactions with the Unfettered are concerned."`
+			
+			label trouble
+			branch "less trouble"
+				"reputation: Hai" >= 1
+				"reputation: Hai (Wormhole Access)" >= 1
+				"reputation: Hai Merchant" >= 1
+				"reputation: Hai Merchant (Human)" >= 1
+			# At least one other group besides the Sympathizers are upset.
+			branch "explained trouble"
+				has "Hai Sympathizer Warning: Explained Trouble"
+			`	"You have gotten yourself in trouble Captain."`
+			label "explained trouble"
+			action
+				"reputation: Hai" >?= 1
+				"reputation: Hai (Wormhole Access)" = "reputation: Hai"
+				"reputation: Hai Merchant" >?= 1
+				"reputation: Hai Merchant (Human)" >?= 1
+			`	He slides a data stick across to you. "Fortunately for you, in this case, I happened to see the event, and have argued that you did not know what you were getting into. This stick contains the security report file which indicates that you have been given a warning... keep this for your records, Hai Security certainly will. There will not be future warnings."`
+				goto end
+			label "less trouble"
+			# Only the Sympathizers are upset.
+			`	"So far, you have been lucky. Hai Security has taken your history into account, and decided your recent activities are within the tolerable range of responses to the hostile situations you have got yourself into..." He slides a data stick across to you. "Nevertheless, this is a copy of the file on you that states you have now been advised of the situation. If you now proceed to upset Hai Security, they will not provide you with a warning."`
+			
+			label end
+			`	The waitress arrives with your food, and the merchant stands up. "Good luck Captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```

Comment:
```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, Captain <last>. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```
or
```suggestion
			`	The waitress arrives with your food, and the merchant stands up. "Good luck, captain. If you get into trouble I recommend picking up some jobs to take aid supplies to the Unfettered, if you can manage that without shooting anyone...`
```
I prefer the first one because this Hai already knows your name so he might as well say it.

Reply:
He's telling you off, and telling you that you just got away with some sketchy business because he's vouched for you being an ignorant outsider this one time.
It's not necessarily politic to keep saying your name where less forgiving folk might overhear.

So no, he might as well not say it.

But he's still referring to you using 'captain' as a title, which still makes it "Captain" with capitalisation.
For both of these you could add the comma on it's own, but I can't accept any of these suggestions as they currently stand.

Reply:
Agreed on capitalised Captain as a title.